### PR TITLE
Fix documentation for URL query string parameters

### LINKED ISSUE #14477: Documentation fixup for URL query string parameters
There are two FAQ entries that are very similar, [here](https://docs.phpmyadmin.net/en/latest/faq.html#can-i-access-directly-to-database-or-table-pages) and [here](https://docs.phpmyadmin.net/en/latest/faq.html#which-parameters-can-i-use-in-the-url-that-starts-phpmyadmin). This could be improved, cross-referenced, and/or even merged.

### PR DESCRIPTION
**Pull request by AkshatSrivastava2**
Fixes #14477 
Signed-off-by: AkshatSrivastava2 <akshatsrivastava2@gmail.com>

Before submitting pull request, please check that every commit:

- [x] Has proper Signed-Off-By
- [x] Has commit message which describes it
- [ ] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [ ] Any new functionality is covered by tests

### CODE REVIEW

File: doc/faq.rst [hunk 1]
```diff
@@ -480,6 +480,8 @@ need mod\_rewrite to be enabled. Then you just need to create
 following :term:`.htaccess` file in root folder of phpMyAdmin installation (don't
 forget to change directory name inside of it):
 
+For more details about this see [Section :ref:`faq4_8`]
```
Comment (Piyush3079) [inline doc/faq.rst]: you can merge code for line 471 and line 1175, 1176 and 1177 as both of them were about how to use `servers` parameter.

File: doc/faq.rst [hunk 2]
```diff
@@ -1179,6 +1181,8 @@ usage of 'cookie' ``auth_type``.
 For example direct login URL can be constructed as
 ``https://example.com/phpmyadmin/?pma_username=user&pma_password=password``.
 
+For more details about this see [Section :ref:`faq1_34`]
```
Comment (Piyush3079) [inline doc/faq.rst]: you can use `For more details about parameters, refer [Section :ref:`faq1_34`]`

File: doc/faq.rst (hunk 1 above)
Comment (AkshatSrivastava2) [reply doc/faq.rst]: I am a bit unclear. Where do I have to merge and put that? Please elaborate.

File: doc/faq.rst (hunk 2 above)
Comment (AkshatSrivastava2) [reply doc/faq.rst]: @Piyush3079  I have made this change. Please make the above doubt clear so that I could send in the updated code.

**Comment by ibennetch** [comment]
@AkshatSrivastava2 Thanks for your interest in contributing to the project! I ended up fixing this and a couple of other grammatical problems with 417c7fc, but we'd love to have you contribute to something else (or this, too, if you think I missed something important).

**Pull request closed without merging**